romancal/regtest/test_tweakreg.py

```diff
@@ -64,3 +66,24 @@ def test_tweakreg(rtdata, ignore_asdf_paths, tmp_path):
         f"DMS280 MSG: Was the proper TweakReg data produced? : {diff.identical}"
     )
     assert diff.identical, diff.report()
+
+    wcstweak = tweakreg_out.meta.wcs
+    orig_model_asdf = asdf.open(orig_uncal)
+    wcstrue = orig_model_asdf["romanisim"]["wcs"]  # simulated, true WCS
```

Comment:
This is the uncal cube with attached 2D WCS, generated manually?
There are different ways to generate a "true" WCS, is the process described somewhere?

Reply:
Every current file that romanisim makes gets a special 'romanisim' extension at the root of the asdf tree.  This extension contains the WCS that was used to simulate the image.  I'm comparing that true WCS (true, in the sense that it was the WCS actually used to simulate the image) to the WCS that tweakreg recovered.

The process used to generate a simulator WCS is the same as the process used for AssignWCS in romancal.  It's just CRDS distortion maps + shifts and scales + rotations on the sky.  The only systematic difference between the simulator WCS and the default AssignWCS WCS in romancal for these files comes from the fact that we have inserted a small error into the wcsinfo in the "shift" files; tweakreg is responsible for finding and correcting this error.

Does that sound right to you?